### PR TITLE
ICDS - changes in KPI, charts, maps and legends.

### PR DESCRIPTION
**Pull request by lwyszomi**
@emord

### CODE REVIEW

File: custom/icds_reports/utils.py [hunk 1]
```diff
@@ -3936,12 +4055,12 @@ def get_data_for(filters):
...
 
     fills = OrderedDict()
-    fills.update({'Launched': GREEN})
+    fills.update({'Launched': PINK})
     fills.update({'defaultFill': GREY})
 
     return [
         {
-            "slug": "awc_daily_statuses",
+            "slug": "awcc_covered",
```
Comment (emord) [inline custom/icds_reports/utils.py]: @lwyszomi Is this correct? I don't see any instances of this elsewhere in the code, but I also don't see any other refs to the value before either.

For now assuming that these are views that have not yet been built

File: custom/icds_reports/views.py [hunk 2]
```diff
@@ -855,10 +864,7 @@ def get(self, request, *args, **kwargs):
 class AWCsCoveredView(View):
     def get(self, request, *args, **kwargs):
         now = datetime.utcnow()
-        month = int(self.request.GET.get('month', now.month))
-        year = int(self.request.GET.get('year', now.year))
-        test_date = datetime(year, month, 1)
-
+        test_date = datetime(now.year, now.month, 1)
```
Comment (emord) [inline custom/icds_reports/views.py]: Why are you removing the ability to change the date? Are these views only used for current metrics?

Comment (emord) [inline custom/icds_reports/views.py]: @lwyszomi bump. I think this is good to go, but want to make sure that this was intentional and not jsut for testing

Comment (lwyszomi) [inline custom/icds_reports/views.py]: @emord Sorry, I was offline on Friday. Yes this intentional, some of the indicators use only location filter and current day/first day of current month.

**Pull request merged**